openshift/scripts/oc_cleanup.sh

```diff
@@ -49,7 +49,12 @@ eval "${OC_CLEAN_FORECAST_CRONJOB}"
 eval "${OC_CLEAN_EC_GDPS_CRONJOB}"
 eval "${OC_CLEAN_EC_HRDPS_CRONJOB}"
 eval "${OC_CLEAN_EC_RDPS_CRONJOB}"
-eval "${OC_DELETE_EC_PODS}"
+
+pods=$(eval "${OC_GET_EC_PODS}")
+if [ -n "$pods" ]; then
+	echo $pods | xargs oc -n ${PROJ_DEV} ${DELETE_OR_GET}
+fi
```

Comment:
How about https://stackoverflow.com/a/8296746/11903963

Reply:
Because that actually stops the bash script from running. Very frustrating, somehow no-one else seems to have that problem. It's what I tried 1st.

Reply:
I see I see. In the comment section it also says it doesn't work in Mac so nvm